play/wikirecent/app/requirements.txt

```diff
@@ -1,4 +1,5 @@
 # Install the Pure Python version of psycopg3 from source
+asyncpg
 git+https://github.com/psycopg/psycopg3.git@4f053f14901454a7c7895b527ddeb9d8ab57d7fe#subdirectory=psycopg3
 tornado
 websockets
```

Comment:
I just noticed that these don't pin the versions at all. That seems a little dangerous? What if one of these packages releases a breaking change? mzbuild probably makes this _ok_ because we won't rebuild the image on every push, but still seems a little dangerous that any change to this directory will pull in the latest version of all these packages?

Reply:
I'll update this to pin versions!